test/fixtures/node_modules/pkgexports/package.json

```diff
@@ -1,6 +1,5 @@
 {
-  "name": "@pkgexports/name",
-  "main": "./asdf.js",
+  "name": "pkg-exports",
```

Comment:
The previous value looked more distinct from the directory name. I had to do a double-take to see the dash. Can we make this `package-json-name-value` or something like it?

Reply:
I switched the test case for `@pkg/name` to the other example, because we needed an example with no main to catch the bug from https://github.com/nodejs/node/issues/30633 since it only applied for non-scope names specifically.

Reply:
The coverage on the scoped package names still applies equivalently for the pkgexports-sugar example though which was why I changed that one to use scoping instead.